### PR TITLE
Removes nested A tag from extensionmanager code

### LINKED ISSUE #13664: HTML validation error in console for extension manager
## Description

```
Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>.
    at a
    at a
    at li
    at p (-node_modules_react-paginate_dist_react-paginate_js-node_modules_semver_index_js.073d91b924a13a8f396f.js:548:3724)
    at ul
    at o (vendors-node_modules_react-paginate_dist_react-paginate_js-node_modules_semver_index_js.073d91b924a13a8f396f.js:548:6557)
    at div
    at div
    at ListView (/packages_extensionmanager_lib_index_js.ee6ace975db8b9f0a079.js:613:13)
```

## Reproduce

1. Open latest dev version of JupyterLab 4.0
2. Open browser console

## Expected behavior

No error

## Context

- Browser and version: Chrome
- JupyterLab version: around 4.0.0a31

### PR DESCRIPTION
**Pull request by JasonWeill**
## References

Fixes #13664.

## Code changes

Removes redundant `a` tag that had been moved to its current location in a change by @fcollonval :

https://github.com/jupyterlab/jupyterlab/blob/c6d57dc28dd566aa60fe95a7548640143fb443ee/packages/extensionmanager/src/widget.tsx#LL240C36-L240C42

(Note that the tag already existed in the code; in #12866, it was merely moved)

Browsers could handle this link; the nested `a` tag was causing errors in the Firefox console, but no apparent loss of functionality.

## User-facing changes

None.

## Backwards-incompatible changes

None.

### CODE REVIEW

**Review by fcollonval — CHANGES_REQUESTED** [review]
Thanks @JasonWeill 

It looks this is polluted by changed related to xterm.js upgrade. Would you mind removing that from this PR?

**Comment by JasonWeill** [comment]
@fcollonval Removed stray changes from #13685.

File: packages/extensionmanager/src/widget.tsx [hunk 1]
```diff
@@ -243,7 +243,7 @@ function ListView(props: ListView.IProperties): React.ReactElement<any> {
           <ReactPaginate
             previousLabel={'<'}
             nextLabel={'>'}
-            breakLabel={<a href="">...</a>}
+            breakLabel='...'
```
Comment (fcollonval) [inline packages/extensionmanager/src/widget.tsx:246]:
```suggestion
            breakLabel="..."
```

**Review by fcollonval — APPROVED** [review]
Thanks

> CI failure are not related

**Pull request merged**